apps/nrm/fields/index.js

```diff
@@ -271,6 +271,7 @@ module.exports = {
     mixin: 'textarea',
     'ignore-defaults': true,
     formatter: ['trim', 'hyphens'],
+    labelClassName: 'visuallyhidden',
```

Comment:
If this field is the heading for the page then I think it's meant to use `isPageHeading:true`

https://github.com/UKHomeOfficeForms/hof/blob/271e6850d3d6ede099e4c442cfbee22ab93f7ac4/sandbox/apps/sandbox/fields.js#L11

here is the hof code

https://github.com/UKHomeOfficeForms/hof/blob/271e6850d3d6ede099e4c442cfbee22ab93f7ac4/frontend/template-mixins/partials/forms/textarea-group.html#L5